fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/FHIRPersistenceJDBCFactory.java

```diff
@@ -27,7 +27,7 @@ public FHIRPersistence getInstance() throws FHIRPersistenceException {
             // each request gets a new instance of the FHIRPersistenceJDBCImpl, sharing
             // the common (tenant-aware) cache object
             FHIRPersistenceJDBCCache cache = tenantCache.getCacheForTenantAndDatasource();
-            return new FHIRPersistenceJDBCImpl(cache);
+            return new FHIRPersistenceJDBCImpl(cache, getPayloadPersistence());
```

Comment:
I want to double check this... 

each payload persistence is in the same datasources PropertyGroup as it is specified today? or is this in a sub propertygroup